crypto/dilithium/pqcrystals_dilithium_ref_common/sign.c

```diff
@@ -68,68 +73,86 @@ int crypto_sign_keypair(ml_dsa_params *params, uint8_t *pk, uint8_t *sk) {
 }
 
 /*************************************************
-* Name:        crypto_sign_signature
+* Name:        crypto_sign_keypair
 *
-* Description: Computes signature.
+* Description: FIPS 204: Algorithm 1 ML-DSA.KeyGen
+*              Generates public and private key.
 *
 * Arguments:   - ml_dsa_params: parameter struct
+*              - uint8_t *pk: pointer to output public key (allocated
+*                             array of CRYPTO_PUBLICKEYBYTES bytes)
+*              - uint8_t *sk: pointer to output private key (allocated
+*                             array of CRYPTO_SECRETKEYBYTES bytes)
+*
+* Returns 0 (success)
+**************************************************/
+int crypto_sign_keypair(ml_dsa_params *params, uint8_t *pk, uint8_t *sk) {
+  uint8_t seed[SEEDBYTES];
+  /* Get randomness for rho, rhoprime and key */
+  RAND_bytes(seed, SEEDBYTES);
```

Comment:
RAND_bytes can never fail, but you should still check the return code so static analysis tools don't complain about not checking the return value.